src/url-replacements.js

```diff
@@ -14,15 +14,16 @@
  * limitations under the License.
  */
 
-import {assert} from '../src/asserts';
-import {documentInfoFor} from '../src/document-info';
-import {parseUrl, removeFragment} from '../src/url';
+import {assert} from './asserts';
+import {documentInfoFor} from './document-info';
+import {getService} from './service';
+import {parseUrl, removeFragment} from './url';
 
 
 /**
  * This class replaces substitution variables with their values.
  */
-export class UrlReplacements {
+class UrlReplacements {
```

Comment:
These were two reasons why I exported the class:
1. Avi was saying that he wanted to customize url replacements for analytics that only applied there.
2. I wanted to keep it extensible for amp-list for infinite scroll. E.g. "https://service/list.json?page=PAGE", where a hypothetical "PAGE" var would only be available in the context of amp-list.